utils/forest_utils/src/proofs_api/paramfetch.rs

```diff
@@ -72,6 +76,28 @@ pub fn set_proofs_parameter_cache_dir_env(data_dir: &Path) {
     std::env::set_var(DIR_ENV, param_dir(data_dir));
 }
 
+/// Ensures the parameter files are downloaded to cache dir
+pub async fn ensure_params_downloaded() -> anyhow::Result<()> {
```

Comment:
Let's remove `CHECKED` and `LOCK`. We can add them back if/when we decide on downloading proof-parameters in the background.